setup.py

```diff
@@ -4,14 +4,22 @@
 # This source code is licensed under the MIT license found in the
 # LICENSE file in the root directory of this source tree.
 
+"""
+    isort:skip_file
+"""
+
 import os.path
 import sys
 
 import setuptools
 
 sys.path.insert(0, os.path.join(os.path.dirname(__file__), "habitat"))
+
 from version import VERSION  # noqa
 
+
```

Comment:
Is BLACK okay with this, this seems like a PEP violiation

Reply:
Seems like it didn't run on setup.py for some reason....